zerver/openapi/zulip.yaml

```diff
@@ -307,6 +315,51 @@ paths:
                                   "setting": true,
                                   "id": 0,
                                 }
+                            - type: object
+                              description: |
+                                Event sent to a user's clients when that user's settings
+                                have changed.
+
+                                **Changes**: New in Zulip 5.0 (feature level 87), replacing the
+                                previous `update_display_settings` event type, which is still
+                                present for backwards compatibility reasons.
+                              properties:
+                                id:
+                                  $ref: "#/components/schemas/EventIdSchema"
+                                type:
+                                  allOf:
+                                    - $ref: "#/components/schemas/EventTypeSchema"
+                                    - enum:
+                                        - user_settings
+                                op:
+                                  type: string
+                                  enum:
+                                    - update
+                                setting_name:
+                                  type: string
+                                  description: |
+                                    Name of the changed setting.
+                                setting:
+                                  description: |
+                                    New value of the changed setting.
+                                  oneOf:
+                                    - type: boolean
+                                    - type: integer
+                                    - type: string
+                                language_name:
+                                  description: |
+                                    Present only if the setting to be changed is
+                                    `default_language`. Contains the name of the
+                                    new default language in English.
+                                  type: string
+                              additionalProperties: false
+                              example:
+                                {
+                                  "type": "user_settings",
+                                  "setting_name": "high_contrast_mode",
+                                  "setting": false,
+                                  "id": 0,
```

Comment:
Do we want to adjust the format for these dictionaries as well?  E.g. I think `/setting/value/` would definitely be an improvement; might be worth doing `s/setting_name/property/` to match the `do_set_realm_property` format.

Reply:
Yes, this would be better. Updated.